site2/docs/functions-overview.md

```diff
@@ -26,9 +26,9 @@ Each time a function receives a message, it completes the following consume-appl
 1. Consumes the message from one or more **input topics**. 
 2. Applies the customized (user-supplied) processing logic to the message.
 3. Publishes the output of the message, including:
-    a) writes output messages to an **output topic** in Pulsar
-    b) writes logs to a **log topic** (if it is configured)for debugging
-    c) writes [state](functions-develop-state.md) updates to BookKeeper (if it is configured) 
+    - writes output messages to an **output topic** in Pulsar
+    - writes logs to a **log topic** (if it is configured)for debugging
+    - writes [state](functions-develop-state.md) updates to BookKeeper (if it is configured) 
```

Comment:
I think the original list is an ordered list?

Reply:
This abc list did not display:
![image](https://user-images.githubusercontent.com/37220920/199180662-1acc358a-9e42-4263-86f8-aa7a274194f2.png)

So change it to an unordered list.

Reply:
Yes. I mean if the original purpose is an ordered list, you may use:

```suggestion
    1. writes output messages to an **output topic** in Pulsar
    2. writes logs to a **log topic** (if it is configured)for debugging
    3. writes [state](functions-develop-state.md) updates to BookKeeper (if it is configured) 
```

Reply:
Make sense